docs/DeveloperGuide.md

```diff
@@ -196,6 +196,58 @@ This is to prevent `FilterCommand` from taking on more responsibilities (Separat
 `FilterCommand` having `setPersonPredicate()` method:
 This is so that `FilterCommand` has the required argument of type `Predicate<Person>` to be used in the `updateFilteredPersonList()` method. Since the `Predicate<Person>` object is created by chaining the multiple predicates, no parsing is involved to create this `Predicate`.
 
+### Sort Feature
+
+##### Implementation
+
+The sort mechanism is facilitated by JavaFX's `SortedList` within ModelManager, `SortCommand` and `SortCommandParser`. `SortCommandParser` extends the types of command parsers in StaffBookParser, and returns a `SortCommand` to be executed by the LogicManager. This execution also updates the `SortedList` in  Model via ModelManager. Additionally, it implements the following operations:
+
+* `SortCommandParser#parse()`  — Parses user input to identify the attribute to be sorted
+* `ModelManager#updateSortedPersonList()` — Update the comparator used by SortedList resulting in the data being sorted accordingly
+
+Given below is an example usage scenario and how the sort mechanism behaves at each step.
+
+Step 1. Initially, the `SortedList` is initialized with a null comparator when the model loads.
```

Comment:
Since this is not shown in the diagram, would it be better to remove this step?

Reply:
Amended